server/athenian/api/__main__.py

```diff
@@ -199,7 +236,8 @@ def _init_sentry(log: logging.Logger, app_env: _ApplicationEnvironment) -> None:
 
     def warn(env_name):
         logging.getLogger(metadata.__package__).warning(
-            "Skipped Sentry initialization: %s envvar is missing", env_name)
+            "Skipped Sentry initialization: %s envvar is missing", env_name
```

Comment:
Wow, there is no comma here. I am pretty sure that another pass of add-trailing-comma + chorny should add it.